modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,22 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parseAndSummarise(file: str, config: Config,
+                        delete_after_parsing: bool):
+    with open(file, 'rb') as fd:
+        data = fd.read()
+        stream = DataStream(data, byte_order='big', is_signed=False)
```

Comment:
I think endianness will be a issue if we are trying to use this tool in the BMC because BMC is little endian.

we need to check the system endianness and read it?

Actually I think it should based on the PEL file writer, not based on the system from where we are running the PEL tool. @spinler any thoughts?

Reply:
PEL data is bit endian, and that is what the arg on DataStream is specifying.